src/renderer/dx/DxFontInfo.cpp

```diff
@@ -0,0 +1,382 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "precomp.h"
+
+#include "DxFontInfo.h"
+
+#include "unicode.hpp"
+
+#include <VersionHelpers.h>
+
+static constexpr std::wstring_view FALLBACK_FONT_FACES[] = { L"Consolas", L"Lucida Console", L"Courier New" };
+
+using namespace Microsoft::Console::Render;
+
+DxFontInfo::DxFontInfo() noexcept :

```

Comment:
Auditing won't let me use the other constructors because they would throw exceptions, and default constructor should not throw exceptions :/

Reply:
ohhhhhhhhhhhhhhhhhhhh yes ok then its fine.